src/cf-forms/src/atoms/text-input.less

```diff
@@ -1,7 +1,7 @@
 .a-text-input {
     .u-webfont-regular();
 
-    // Browser Reset
```

Comment:
This is not a sentence, or even a phrase that implies a full sentence :) If you want to be consistent with the periods, rephrase.